digitalpy/core/component_management/__init__.py

```diff
@@ -0,0 +1,2 @@
+"""This core package contains functions related to Managment of components, this includes discovery, 
+installation and deinstallation"""
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>W292 no newline at end of file